cli/cache/incremental.rs

```diff
@@ -242,22 +241,20 @@ fn create_tables(
   conn: &Connection,
   cli_version: String,
 ) -> Result<(), AnyError> {
-  // INT doesn't store up to u64, so use TEXT
-  conn.execute(
-    "CREATE TABLE IF NOT EXISTS incrementalcache (
-        file_path TEXT PRIMARY KEY,
-        state_hash TEXT NOT NULL,
-        source_hash TEXT NOT NULL
-      )",
-    [],
-  )?;
-  conn.execute(
-    "CREATE TABLE IF NOT EXISTS info (
-        key TEXT PRIMARY KEY,
-        value TEXT NOT NULL
-      )",
-    [],
-  )?;
+  let query = format!(
+    "{INITIAL_PRAGMAS}
+  -- // INT doesn't store up to u64, so use TEXT
```

Comment:
Nitpick: Now this comment is part of the binary. Can you move it back out and just make it more explanatory (ex. `// INT doesn't store up to u64, so use TEXT for source_hash`)?